content/400-pulse/200-getting-started.mdx

```diff
@@ -23,6 +23,7 @@ You will also need a database with the following configurations:
 - Ensure your database is publicly accessible.
 - [Set the `wal_level` setting in PostgreSQL to `logical`](/pulse/getting-started#wal_level).
```

Comment:
Is this line supposed to be a link? It links me to the same page

Reply:
It links to the section heading `wal_level, it's supposed to take you there. Seems broken.